index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-First</title>
+
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/nav.css">
+  <link rel="stylesheet" href="styles/info.css">
+  <link rel="stylesheet" href="styles/footer.css">
+</head>
+<body>
+  <main>
+    <header id="header">
+      <div class="header-container">
+        <p class="text-gradient">Welcome</p>
+        <img class="menu" src="img/menu.svg" alt="menu">
```

Comment:
- [ ] Please use the same hamburger as the original design and you should use a the `nav` tag for the menu hamburger

- Original design 👇 
![image](https://user-images.githubusercontent.com/75126481/219378941-e3d3c106-4abe-4afa-a2ff-8da57b89c397.png)

- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/219379201-0c21eca2-b257-49f7-8617-fa3c030a389e.png)